### PR TITLE
[CodingStyle] Allow Closure and Inner function on FuncGetArgsToVariadicParamRector

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: rules-tests/CodingStyle/Rector/ClassMethod/FuncGetArgsToVariadicParamRector/Fixture/some_closure.php.inc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    };
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\CodingStyle\Rector\ClassMethod\FuncGetArgsToVariadicParamRector\Fixture;
+
+function run()
```
Comment (samsonasik) [inline rules-tests/CodingStyle/Rector/ClassMethod/FuncGetArgsToVariadicParamRector/Fixture/some_closure.php.inc:21]: It seems cause function parent of closure changed as well: 

```diff
-function run()
+function run(...$args)
```

Comment (samsonasik) [reply rules-tests/CodingStyle/Rector/ClassMethod/FuncGetArgsToVariadicParamRector/Fixture/some_closure.php.inc:21]: Fixed 🎉

**Comment by samsonasik** [comment]
handle inner function as well.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

**Comment by TomasVotruba** [comment]
Thank you :clap:

**Pull request merged**